### ISSUE TITLE
Laravel 5.4 Support

### ISSUE THREAD

**Issue by reganjohnson** [Torann/laravel-geoip#73]
Not working with 5.4

**Comment by reganjohnson** [comment]
In `GeoIPServiceProvider.php` change the `share()` cases to be `singleton()`.

```
$this->app->singleton('geoip', function($app)
{
	return new GeoIP($app['config'], $app["session.store"]);
});

$this->app->singleton('command.geoip.update', function ($app)
{
	return new UpdateCommand($app['config']);
});
```

**Comment by Torann** [comment]
In the next two days, I'm going to be updating all of my packages.

**Comment by Torann** [comment]
It looks like you are v0.2, you need to update to v1.0

http://lyften.com/projects/laravel-geoip/doc/upgrade.html

**Comment by Torann** [comment]
I just pushed a fix for v0.2.

Version 0.2 will no longer be maintained after Laravel 5.4.
